integration_test/test_xgboost_experiment.py

```diff
@@ -104,6 +104,19 @@ def test_experiment(self, task):
     self._verify_metrics_for_all_runs(sigopt_suggested_runs, experiment.metrics)
     experiment.archive()
 
+  def test_early_stopping(self):
+    self._form_random_experiment_config('binary')
+    self.experiment_params['early_stopping_rounds'] = 1
+    experiment = sigopt.xgboost.experiment(**self.experiment_params)
+    assert experiment.is_finished()
+    sigopt_suggested_runs = list(experiment.get_runs())
+    for ssr in sigopt_suggested_runs:
+      assert ssr.assignments['early_stopping_rounds'] == self.experiment_params['early_stopping_rounds']
+      for param in experiment.parameters:
+        if param.name == 'num_boost_round':
```

Comment:
Does this for loop and if statement need to exist? If early stopping happens, that's regardless of whether we are setting `num_boost_round` as a tuning parameter or not, no?